.github/workflows/e2e_additional_editor_and_player.yml

```
@@ -55,4 +55,5 @@ jobs:
           name: protractor-video
           path: /home/runner/work/oppia/protractor-video
       - name: Checking Memory Usage
+        if: {{ always() }}
```

```suggestion
        if: ${{ always() }}
```

I think. ditto elsewhere

---

Done